.github/workflows/update-openssl.yml

```diff
@@ -0,0 +1,70 @@
+name: OpenSSL update
+on:
+  schedule:
+    # Run once a week at 00:05 AM UTC on Sunday.
+    - cron: 5 0 * * 0
+
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+jobs:
+  openssl-update:
+    if: github.repository == 'nodejs/node'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          persist-credentials: false
+      - name: Check if update branch already exists
+        run: |
+          BRANCH_EXISTS=$(git ls-remote --heads origin actions/tools-update-openssl)
```

Comment:
I think this block the automatic backport. We won't be able to update v14 and v16 at same time considering the race condition of the branch creation. 

So, I'll include the labels `dont-land-on-vX` and after landing you will need to manually backport and change the branch name to `actions/tools-update-openssl-vX`.

Does that make sense to you?

Reply:
This is the same logic used for the other deps. See this line from the current dependency update action: https://github.com/nodejs/node/blob/9b1ed04f2f0cbe74dbbfa963a8fcc7f5a5148bf9/.github/workflows/tools.yml#L153

As far as I understand, this would be already an issue for `corepack`, `eslint`, etc right? Since the branches created for them are called `tools-update-corepack` and `tools-update-eslint` without mentioning the Node version.

Do you know how those are handled currently?

Reply:
I don't think we have different versions of eslint/corepack in each release line. Could you check if the bot has ever opened PR targeting other branches instead of `main`?

Reply:
The bot will not because the automatic scheduled runs only happen on the default branch: 
https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule
> Scheduled workflows run on the latest commit on the default or base branch.